src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/eksportering/SykefraværsstatistikkTilEksporteringRepository.java

```diff
@@ -64,73 +66,63 @@
         }
     }
 
-    @Deprecated
+    /* Sykefraværsprosent Næring */
+
     public List<SykefraværsstatistikkNæring> hentSykefraværprosentAlleNæringer(
-            ÅrstallOgKvartal årstallOgKvartal) {
-        try {
-            return namedParameterJdbcTemplate.query(
-                    "select naring_kode, arstall, kvartal, antall_personer, tapte_dagsverk, mulige_dagsverk " +
-                            "from sykefravar_statistikk_naring " +
-                            "where arstall = :arstall and kvartal = :kvartal order by arstall, kvartal, naring_kode",
-                    new MapSqlParameterSource()
-                            .addValue("arstall", årstallOgKvartal.getÅrstall())
-                            .addValue("kvartal", årstallOgKvartal.getKvartal()),
-                    (rs, rowNum) -> mapTilSykefraværsstatistikkNæring(rs)
-            );
-        } catch (EmptyResultDataAccessException e) {
-            return Collections.emptyList();
+            ÅrstallOgKvartal sisteÅrstallOgKvartal,
+            int antallKvartaler
+    ) {
+        if (antallKvartaler < 2) {
```

Comment:
Den er ikke feil, men blir ikke litt mer forståelig om vi sier `antallKvartaler == 1`

Reply:
Jeg tror den ble/skulle bli brukt for eksportering av bransje/næring som vi ikke implementert ennå (pga endret planen og fokus på hva team PIA trenger). Så det stemmer at den er død kode per i dag. 
Så hva tenker dere er best å gjøre? 
For min del, er den ikke en stor eller fansy kode og vi kan fjerne død kode og den blir skrevet igjen om behovet blir der igjen.